### PR TITLE
Create free.json

### PR DESCRIPTION
**Pull request by slimepointe**

### CODE REVIEW

**Review by phenax — CHANGES_REQUESTED** [review]
This seems like an incorrect use of the service. Please go through the docs or create an issue if you need any help.

File: domains/free.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "my gamer page",
+  "repo": "https://github.com/freegocrazy",
+  "owner": {
+    "username": "freegocrazy",
+    "email": "freeisverycool@gmail.com"
+  },
+  "record": {
+    "TXT": "whats poppin!"
```
Comment (MaskDuck) [inline domains/free.json:9]: I would like to see an actual DNS record, not a random string here.

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by SX-9** [comment]
He/she is probaply new to this kind of stuff

**Comment by MaskDuck** [comment]
mm

**Pull request closed without merging**